main.js

```diff
@@ -4,22 +4,39 @@ var bodyInput = document.querySelector('.form-textarea-body');
 var submit = document.querySelector('.form-button-save');
 var cardSection = document.querySelector('.section-cards');
 var form = document.querySelector('.form');
-var allCards = [];
 var buttonOpen = document.querySelector('.nav-button-open');
 var navSection = document.querySelector('.nav-section');
 var image = document.getElementById('menu');
+var allCards = [];
 
 submit.addEventListener('click', submitNewIdea);
+titleInput.addEventListener('keyup', enableButton);
+bodyInput.addEventListener('keyup', enableButton);
+window.onload = loadCards;
+cardSection.addEventListener('click', function() {
+  cardEvent(event.target);
```

Comment:
where is event coming form here? since it's not a param in the callback function. Also, try to swap to arrow function to be consistent